roles/openstack-setup/tasks/cinder.yml

```diff
@@ -19,6 +19,15 @@
     - (item.name in ['cinder','cinderv2','cinderv3'])
   run_once: true
 
+# FIXME: this task should be removed once old service is disabled for all envs
```

Comment:
Different with nova, cinder doesn't provide an API of deleting service.
Disabling is not as perfect as deleting, but it's also fine.